src/main/java/it/pagopa/interop/probing/eservice/operations/model/view/EserviceView.java

```diff
@@ -73,8 +68,13 @@ public class EserviceView implements Serializable {
   @Column(name = "version_number")
```

Comment:
missing `@Version`?

Please use a class abstract called abstractEntity that extends a interface Entity that include all fields as version that are commons to each entity

Reply:
yes, given that it is a view, and is therefore read-only, I thought it wasn't necessary, do you agree? eventually we will insert it